site/content/docs/5.2/examples/form/index.html

```diff
@@ -0,0 +1,237 @@
+---
+layout: examples
+title: Form example
+extra_js:
+- src: "form.js"
+---
+
+<header class="sticky-top">
+  <nav class="navbar navbar-dark navbar-expand-lg bg-dark supra" aria-label="Supra navigation">
+    <div class="container-xxl">
+      <ul class="navbar-nav ms-auto">
+        <li class="nav-item"><a class="nav-link active" href="#" aria-label="EN English version" aria-current="true">EN</a></li>
+        <li class="nav-item"><a class="nav-link" href="#" aria-label="FR Version française" lang="fr" hreflang="fr">FR</a></li>
+      </ul>
+    </div>
+  </nav>
+  <nav class="navbar navbar-dark bg-dark navbar-expand-lg" aria-label="Global navigation">
+    <div class="container-xxl">
+      <div class="navbar-brand me-auto me-lg-4">
+        <a class="stretched-link" href="#">
+          <img src="/docs/{{< param docs_version >}}/assets/brand/orange-logo.svg" width="50" height="50" alt="Boosted - Back to Home"
+            loading="lazy">
+        </a>
+        <h1 class="title">Travel</h1>
+      </div>
+      <button class="navbar-toggler collapsed" type="button" data-bs-toggle="collapse" data-bs-target=".global-header-1"
+        aria-controls="global-header-1.1 global-header-1.2" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div id="global-header-1.1" class="navbar-collapse collapse me-lg-auto global-header-1">
+        <ul class="navbar-nav">
+          <li class="nav-item"><a class="nav-link" href="#">All inclusive cards</a></li>
+          <li class="nav-item"><a class="nav-link" href="#">Data only cards</a></li>
+          <li class="nav-item"><a class="nav-link" href="#">Top up</a></li>
+          <li class="nav-item"><a class="nav-link" href="#">Assistance</a></li>
+        </ul>
+      </div>
+      <div id="global-header-1.2" class="navbar-collapse collapse d-sm-flex global-header-1">
+        <ul class="navbar-nav flex-row">
+          <li class="nav-item">
+            <a href="#" class="nav-link nav-icon">
+              <svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false"
+                class="overflow-visible">
+                <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#buy"></use>
+              </svg>
+              <span class="visually-hidden">Basket</span>
+            </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link nav-icon">
+              <svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false"
+                class="overflow-visible">
+                <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#user"></use>
+              </svg>
+              <span class="visually-hidden">My account</span>
+            </a>
+          </li>
+        </ul>
+        <ul class="navbar-nav border-top border-1 border-dark d-flex flex-row d-lg-none">
```

Comment:
This part should be inserted right after the navbar links so l.37 + have `aria-current="true"`